ngraph/test/CMakeLists.txt

```diff
@@ -450,6 +450,7 @@ target_link_libraries(unit-test PRIVATE ie_backend)
 
 if (NGRAPH_ONNX_IMPORT_ENABLE)
     target_link_libraries(unit-test PRIVATE onnx_importer)
+    target_link_libraries(unit-test PRIVATE onnx_editor)
```

Comment:
let's merge these 2 lines

Reply:
in current state (onnx_editor have no separate control flag yet) sure, done